LiskHQ/lisk-mobile
# Adapt the error statement with the new color scheme - Closes #991

# What was the bug or feature?
It is described at #991 

### How did I fix it?
I added error to empty field for address field in addBookmark and Recipient screens.

## Type of change
- Enhancement (a non-breaking change which adds functionality)

### How to test it?
Go to addBookmark screen, start inserting address and erase so the field is empty, the error should be display below the field. 
Go to recipiente screen, repeat the procedure, the error should be displayed at the top of the screen. 

QA Accs:
- Errors should match new designs that can be found at [inVision](https://projects.invisionapp.com/d/main#/projects/prototypes/18683515).
- Main changes are: 
1. The error colors changed slightly to (#EC6868). 
2. In add Bookmark screen, errors were displayed in redish text with icon, now they are displayed in redish block, the text should be white and the icon is gone. Colors are the same for light and dark mode.
3. In send screen the only change is the color as mentioned in item 1.

# Checklist:
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
